content/en/network_performance_monitoring/guide/detecting_a_network_outage.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Detecting a Network Outage
+kind: guide
+---
+Network outages are often disguised as infrastructure, application, or container issues, which makes them hard to detect. Without visibility into your regional network performance or that of third-party endpoints you rely on, it may take hours to detect an outage, which could ultimately affect your customers and the bottom line. 
+
+Alternatively with NPM, you can detect network outages in minutes by using network flow data alongside process metrics, traces, logs, and infrastructure metrics, you don’t have to make assumptions about the root of an issue - you can use process of elimination (see the steps below) to determine whether you’re experiencing a network outage.
+
+## Traffic overloading the underlying infrastructure 
+
+Use NPM metrics to see whether your source endpoint may be sending an enormous amount of traffic or making a large number of open connections to the destination endpoint. When selecting a faulty dependency (e.g. with high latency), the sidepanel graphs can be used to spot such spikes in traffic, which may overwhelm your receiving application to the point that it cannot (in the case of TCP) respond to all connections, leading to increased packet loss and thus, TCP round-trip time. 
+
+{{< img src="network_performance_monitoring/guide/detecting_a_network_outage/npm-metrics.png" alt="Traffic overloading of the underlying infrastructure">}}
+
+## CPU overconsumption of the underlying infrastructure
+
+On the other hand, resource overconsumption of either the client or server endpoint could be the culprit of poor communication between the two. In the sidepanel Processes tab, we can scope our view to processes running on either the source or destination to spot any heavy software that may be degrading the performance of the underlying hosts or containers, thus leading to their inability to respond to network calls. In this case, it’s not enough that an underlying host is running hot to identify the root of the application latency, but why it is running hot. Grouping your process metrics by command gives you this granularity, since you can identify what software is eating up your CPU. 
+
+{{< img src="network_performance_monitoring/guide/detecting_a_network_outage/processes.png" alt="CPU overconsumption of the underlying infrastructure">}}
+
+## Application errors in code
+
+Network errors and latency can also be caused by client-side application errors. For instance, if your application is generating connections on loop unnecessarily, it could be overwhelming the endpoints that rely on it, leading to downstream application and network issues. Use the `Traces` tab in NPM or the `Network` tab in APM Traces to determine whether this is the case.
```

Comment:
```suggestion
Network errors and latency can also be caused by client-side application errors. For instance, if your application is generating connections on loop unnecessarily, it could be overwhelming the endpoints that rely on it, leading to downstream application and network issues. Use the `Traces` tab in NPM or the `Network` tab in APM Traces and look for application request errors to determine whether this is the case.
```

Reply:
we can also link https://www.datadoghq.com/blog/apm-npm-application-debugging/ for this section